src/plotly/plotly_types.nim

```diff
@@ -197,6 +204,13 @@ type
   RangeSlider* = ref object
     visible*: bool
 
+  # tuple types to set location of subplots within a plot
+  # given in relative coordinates of the plot [0, 1] canvas
+  Domain* = tuple
+    left, bottom, width, height: float
+  DomainAlt* = tuple
```

Comment:
can you document DomainAlt? I'm not sure what it is/does.

Reply:
`DomainAlt` simply refers to the alternate representation of a domain, which uses `right` and `top` instead of `width` and `height`. I simply introduced these as two named types to clear up the proc signatures.
https://github.com/brentp/nim-plotly/blob/6a897fa35ad1291da1c43140bd81655feed1563f/src/plotly/plotly_subplots.nim#L4-L15
then is used to convert `DomainAlt` to a normal `Domain`. I should maybe add a comment to the `DomainAlt` line though (and now that I look at the code for `convertDomain`, use `Domain` in the type check there. Will do both tomorrow.